packages/@aws-cdk/aws-rds/lib/serverless-cluster.ts

```diff
@@ -580,14 +586,21 @@ export class ServerlessCluster extends ServerlessClusterNew {
       throw new Error('Cannot add multi user rotation for a cluster without VPC.');
     }
 
-    return new secretsmanager.SecretRotation(this, id, {
-      ...options,
+    // Rotation options with defaults
+    const rotationOptions: Complete<RotationMultiUserOptions> = {
+      automaticallyAfter: options.automaticallyAfter,
+      endpoint: options.endpoint,
       excludeCharacters: options.excludeCharacters ?? DEFAULT_PASSWORD_EXCLUDE_CHARS,
+      secret: options.secret,
+      vpcSubnets: options.vpcSubnets ?? this.vpcSubnets,
+    };
```

Comment:
This block is repeated 6 times (yes, I know `secret` is not always there 😛). Can we get rid of this duplication?